mrp_progress_button/models/mrp_production.py

```diff
@@ -18,3 +18,12 @@ def action_progress(self):
             }
         )
         return True
+
+    def action_confirm(self):
```

Comment:
You are overriding standard `action_confirm` method in production.order. Use a different name.

Reply:
maybe call super before?